collect_app/src/main/res/layout/url_widget_answer.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/url_button"
+        style="@style/Widget.Collect.Button.Custom"
```

Comment:
Could we consider introducing a style that includes all of the options below so that if we want to change the in-form buttons we can do it in one place?

Reply:
I'd be interested to see if we can get rid of the button... We're hitting the geowidgets next so might make sense to introduce button styles there if they're hanging around.

Reply:
That makes sense, @seadowg. Up to y'all to decide whether you want to add do a style with the next set of widgets or not.